profile_library/shelly/Shelly 1/model.json

```diff
@@ -0,0 +1,26 @@
+{
+  "measure_description": "The measured device fw: 1.14.1-rc1, Eco Mode enabled, Wifi Client enabled. The result represents the average value of measurements collected from the measurement device API for the following scenarios: dumb device/baseline consumption (~4.9W), the the relay turned off, and the relay turned on.",
+  "measure_method": "manual",
+  "measure_device": "Shelly PM Mini gen3",
+  "measure_device_firmware": "20241011-114503/1.4.4-g6d2a586",
+  "measure_settings": {
+    "SAMPLE_COUNT": 200,
+    "SLEEP_TIME": 0.5
+  },
+  "name": "Shelly 1",
+  "aliases": [
+    "SHSW-1"
+  ],
+  "discovery_by": "entity",
+  "device_type": "smart_switch",
+  "only_self_usage": false,
+  "calculation_strategy": "fixed",
+  "standby_power": 0.376,
+  "standby_power_on": 0.618,
+  "sensor_config": {
+    "power_sensor_naming": "{} Device Power",
+    "energy_sensor_naming": "{} Device Energy"
+  },
```

Comment:
```suggestion
```

This can be removed, also matching the docs.
We differentiate the sensor naming to prevent conflicts with the built-in energy meter in the plug, also to make it clear the power is only for the device itself.
When `only_self_usage` is false it makes sense to omit the alternative naming as powercalc will provide the only power sensor.